lib/logflare_web/router.ex

```diff
@@ -146,6 +146,10 @@ defmodule LogflareWeb.Router do
     plug(LogflareWeb.Plugs.VerifyApiAccess, scopes: ~w(partner))
   end
 
+  pipeline :block_system_source do
+    plug(LogflareWeb.Plugs.BlockSystemSource)
```

Comment:
we could add this plug to `:otlp_api` and `:api` without needing a separate pipeline